Detectors/TRD/base/src/TrackletTransformer.cxx

```diff
@@ -58,6 +58,9 @@ float TrackletTransformer::calculateDy(int detector, int slope)
   double padWidth = mPadPlane->getWidthIPad();
 
   float vDrift = mCalVdriftExB->getVdrift(detector);
+  if (vDrift == 0.) {
```

Comment:
why would `vDrift` be 0 actually?

Reply:
On my smallish (100ctfs) dataset I test upon, I saw this line after fitting:
![image](https://user-images.githubusercontent.com/42321018/195551825-c5f9ccc6-c033-4ead-8c98-c6544311b713.png)
From this chamber: 
![image](https://user-images.githubusercontent.com/42321018/195552364-f209562c-7616-4ee7-96f8-8a16532ffe5a.png)
Ran it like this:
![image](https://user-images.githubusercontent.com/42321018/195554133-0b8d9724-22ac-4a76-ab34-9c25a2029da3.png)


Hence I thought that maybe checking for this, might be worthwhile before we throw we an exception.
I did not investigate why this really happens to fit exactly zero here...

Reply:
Ok, so the fit might yield zero. Then I would suggest we change in the calibrator the range for the fit parameter such the minimum value is something like 0.01 or so. We might have a chamber where the HV is strongly reduced or OFF. Then we of course don't want to put just the default vDrift value, but the minimum one instead. And then we can remove this check, or, in case `vDrift` is too close to zero just throw an error

Reply:
I removed the check and set the fit range to minimum of 0.01.

On another note. Just a thought I had, should we maybe prepare some code to store which chamber has what status e.g. HV reduced, HV off, Capacitor 4.7/removed etc.? Then we could handle these chambers differently in calibration code and it would be easier to see why a chamber behaved badly.